src/zenhub.py

```diff
@@ -110,91 +196,67 @@ def _update_issue_pipeline(self, pipeline, pos=None):
         else:
             logger.info(f'{pipeline} is not a valid pipeline.')
 
-    def _update_issue_to_epic(self):
+    def update_issue_to_epic(self):
         # Change the issue into an Epic.
```

Comment:
If you rename the `def` to `promote_issue_to_epic` or something more explanatory you could skip that comment.

Reply:
agreed that the comment isn't necessary